### ISSUE TITLE
Finetune Compressor

### ISSUE THREAD

**Issue by Wangchentong** [amyxlu/plaid#6]
Hi Amy, great work on the plaid paper!

I'm curious if you explored adding a specific loss function to normalize the encoder's output. While the tanh layer is a good approach, it doesn't necessarily guarantee uniform scale and variance across all channels, which could be important for subsequent latent diffusion or flow models.

An ablation study analyzing the compressor's impact on generation performance would be highly valuable and much appreciated!

**Comment by Wangchentong** [comment]
and a side issue, have you tried use VAE as the  compressor?

**Comment by amyxlu** [comment]
Great points! I didn't explore adding a KL to a normal dist. and related normalization tricks when training CHEAP because when I looked at the post-tanh output values as a histogram, things already looked fairly Gaussian (I think there's a figure in the appendix somewhere about this). Also, the main motivation for using the tanh was to bound things between -1 and 1 so that when we do diffusion sampling at each iteration, we can clip the values, as is done in the DDPM and related image diffusion papers. But you raise a good point and in retrospect it wouldn't have hurt (though I'm also not super sure how much it would improve performances either -- if you'd like to add a PR to CHEAP and/or let me know the experiment results if you try it, that would be super cool)

**Comment by Wangchentong** [comment]
Good to know Amy! yes i am trying some new loss and tricks to the compressor to see if we can get a more meaningful and smooth latent space more easily to diffusion with, though from human perspective CHEAP outputs distribution is already good enough. I'll follow up with an email later if i get any intereting result.
